File: app/src/main/java/example/com/archtest/PreferencesManager.java
package example.com.archtest;

import android.content.Context;
import android.content.SharedPreferences;
import android.preference.PreferenceManager;
import android.support.annotation.NonNull;

public abstract class PreferencesManager {
	private static String KEY_LAST_PAGE = "last_loaded_page";

	public static void saveLoadedPage(@NonNull Context context, int pageNumber) {
		SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
		preferences.edit().putInt(KEY_LAST_PAGE, pageNumber).apply();
	}

	public static int getLoadedPage(@NonNull Context context) {
		SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
		return preferences.getInt(KEY_LAST_PAGE, 0);
	}
}

File: app/src/main/java/example/com/archtest/PreferencesManager.kt
package example.com.archtest

import android.content.Context
import android.preference.PreferenceManager

class PreferencesManager {
    companion object {
        val KEY_LAST_PAGE = "last_loaded_page"
        fun saveLoadedPage(context: Context, pageNumber: Int) {
            val preferences = PreferenceManager.getDefaultSharedPreferences(context)
            preferences.edit().putInt(KEY_LAST_PAGE, pageNumber).apply()
        }

        fun getLoadedPage(context: Context): Int {
            val preferences = PreferenceManager.getDefaultSharedPreferences(context)
            return preferences.getInt(KEY_LAST_PAGE, 0)
        }
    }
}